python-checks/src/main/java/org/sonar/python/checks/FloatingPointEqualityCheck.java

```diff
@@ -94,15 +108,41 @@ private boolean isBinaryOperationWithFloat(Expression expression) {
     return false;
   }
 
-  private static PythonQuickFix createQuickFix(BinaryExpression binaryExpression, String operator) {
+  private PythonQuickFix createQuickFix(BinaryExpression binaryExpression, String operator) {
     String notToken = "!=".equals(operator) ? "!" : "";
-    String message = String.format(QUICK_FIX_MESSAGE, notToken);
-    String mathIsCloseText = String.format(QUICK_FIX_REPLACE, notToken,
+    String isCloseModuleName = getModuleNameOrAliasForIsClose();
+    String message = String.format(QUICK_FIX_MESSAGE, notToken, isCloseModuleName);
+    Builder quickFix = PythonQuickFix.newQuickFix(message);
+    
+    String quickFixText = "math".equals(isCloseModuleName) ? QUICK_FIX_MATH : QUICK_FIX_IMPORTED_MODULE;
+    String quickFixTextWithModuleName = String.format(quickFixText, notToken, isCloseModuleName,
         TreeUtils.treeToString(binaryExpression.leftOperand(), false),
         TreeUtils.treeToString(binaryExpression.rightOperand(), false));
 
-    return PythonQuickFix.newQuickFix(message)
-        .addTextEdit(TextEditUtils.replace(binaryExpression, mathIsCloseText))
-        .build();
+    quickFix.addTextEdit(TextEditUtils.replace(binaryExpression, quickFixTextWithModuleName));
+
+    if (importedModuleForIsClose == null) {
+      quickFix.addTextEdit(TextEditUtils.insertAtPosition(0,0, "import math\n"));
+    }
+
+    return quickFix.build();
+  }
+
+  private String getModuleNameOrAliasForIsClose() {
+    if(importedAlias != null){
+      return importedAlias.name();
+    }
+    if(importedModuleForIsClose != null){
+      return importedModuleForIsClose;
+    }
+    return "math";
+  }
+
+  private void addImportedName(AliasedName aliasedName) {
+    List<Name> importNames = aliasedName.dottedName().names();
+    if (importedModuleForIsClose == null && importNames.size() == 1 && SUPPORTED_IS_CLOSE_MODULES.contains(importNames.get(0).name())) {
```

Comment:
Why the condition on `importNames.size() == 1 ` ?
It's unlikely that someone does `import numpy, math`, but who knows?
Also, the current logic will use whatever was imported last, maybe we can have some preference ordering? Although that point is pretty minor.

Reply:
Yes while doing this I noticed that  `dotteName().names()` is always size 1 even with the test `import some_module, math` both imports are considered separately, maybe we could change the API here but I will investigate that later on. Still, I have changed the function to be a bit more robust. For the priority I find it difficult to choose this is why I arbitrarily picked the first one. I could prioritize torch or numpy before math.